runtime/fiber.c

```diff
@@ -183,7 +183,7 @@ Caml_inline struct stack_info* alloc_for_stack (mlsize_t wosize)
   // 2Mb (= extra_size)
   // -------------------- <- [stack], returned from [mmap], page-aligned
   char* stack;
-  stack = caml_mem_map(len + stack_extra_size_for_mmap, 0);
+  stack = caml_mem_map(len + stack_extra_size_for_mmap, 0, "stack");
```

Comment:
Would be nice if this said whether it was the main OCaml stack or a thread stack, with the tid in the latter case.

Reply:
I put the TID in, which sounds useful. Threading the bit for whether it's the main thread is more awkward and seems less helpful (main is when TID = PID, so it's not hard to find out), so I left it at that.